aws-sqs-http-connector/main.go

```diff
@@ -120,6 +126,8 @@ func (conn awsSQSConnector) responseHandler(queueURL string, response string, me
 			)
 			return false
 		}
+	} else {
+		conn.logger.Info(response)
```

Comment:
nitpick, can we have log in this format
```suggestion
		conn.logger.Debug("response received", zap.String("response", response))
```